edit/ui.cattle.io.navlink.vue

```diff
@@ -0,0 +1,395 @@
+<script>
+import { mapGetters } from 'vuex';
+import { isNull, isUndefined } from 'lodash';
+import CreateEditView from '@/mixins/create-edit-view';
+import { SERVICE } from '@/config/types';
+import { PROTOCOLS } from '@/config/schema';
+import CruResource from '@/components/CruResource';
+import LabeledInput from '@/components/form/LabeledInput';
+import RadioGroup from '@/components/form/RadioGroup';
+import FileImageSelector from '@/components/form/FileImageSelector';
+import NameNsDescription, { normalizeName } from '@/components/form/NameNsDescription';
+import Tab from '@/components/Tabbed/Tab';
+import Tabbed from '@/components/Tabbed';
+import LabeledSelect from '@/components/form/LabeledSelect.vue';
+
+const LINK_TYPE_URL = 'url';
+const LINK_TYPE_SERVICE = 'service';
+const LINK_TARGET_NAMED = 'LINK_TARGET_NAMED';
+const LINK_TARGET_BLANK = '_blank';
+const LINK_TARGET_SELF = '_self';
+
+export default {
+  mixins:     [CreateEditView],
+  components: {
+    CruResource,
+    LabeledInput,
+    RadioGroup,
+    NameNsDescription,
+    Tabbed,
+    Tab,
+    FileImageSelector,
+    LabeledSelect
+  },
+  data() {
+    const t = this.$store.getters['i18n/t'];
```

Comment:
Do we need the getter here? We should have access to i18n via `this.t()`.

Reply:
That was the previous implementation, but it returned errors. I asked around, but it seems like for some reasons the compiler still refuses to accept it. Do you have any suggestion in mind which may cause it?

Reply:
It gave me `Cannot read properties of undefined (reading 'length')`, but removing the getter it solves it :sweat_smile: 
Thanks!